cumulus/client/consensus/aura/src/collators/mod.rs

```diff
@@ -43,14 +43,14 @@ pub mod basic;
 pub mod lookahead;
 pub mod slot_based;
 
-// This is an arbitrary value which is likely guaranteed to exceed any reasonable
-// limit, as it would correspond to 30 non-included blocks.
+// This is an arbitrary value which is guaranteed to exceed the required depth for 500ms blocks
+// built with a relay parent offset of 1.
```

Comment:
I would include the math you did in the PR description, here in Rust doc.

Thus, number is not completely arbitrary, it is based on the calculations you did in the PR description which resulted in `37`..?